tests/cpgrid/distribution_test.cpp

```diff
@@ -485,6 +485,9 @@ init_unit_test_func()
 int main(int argc, char** argv)
 {
     Dune::MPIHelper::instance(argc, argv);
+    MPI_Errhandler errhandler;
```

Comment:
missing HAVE_MPI protection is seems, see https://ci.opm-project.org/job/opm-grid/103/